app/models/calculator/flat_percent_item_total.rb

```diff
@@ -11,7 +11,8 @@ class FlatPercentItemTotal < Spree::Calculator
     localize_number :preferred_flat_percent
 
     validates :preferred_flat_percent,
-              numericality: { message: :calculator_preferred_value_error }
+              numericality: { message: :calculator_preferred_value_error },
+              unless: -> { Spree::Config.enable_localized_number? }
```

Comment:
I thought we should incorporate this validation into `localize_number`, and tried, but it's complicated (it's also used slightly differently by `Variant`). 

I think this will be manageable as it is 👍